### ISSUE TITLE
Possibility to set link rel on ACP

### ISSUE THREAD

**Issue by andreadragotta** [NodeBB/NodeBB#8970]
It can be useful tu set in ACP the link rel attribute.

At the moment all links are nofollow and is bad for SEO

**Comment by barisusakli** [comment]
I think this is controlled by markdown plugin.

![image](https://user-images.githubusercontent.com/2583394/100467228-1f1fa400-30a0-11eb-9408-50821bfa4151.png)

Try turning that setting off?

**Comment by julianlam** [comment]
Yes, that's right -- Markdown adds the `rel` tag. Please be warned that `rel="nofollow"` for a reason. Spammers post spam on forums specifically for backlinking purposes.

**Comment by oplik0** [comment]
`rel="nofollow"` was actually recommended for SEO for links that you don't trust or don't necessarily want to associate your site with. This usually included user generated content which is what forums are made of.

In 2019 and 2020, however, Google changed things up a little bit by adding `ugc` and `sponsored` attributes and starting to treat `nofollow` as a hint rather than actual rule (so the crawlers will sometimes follow this link).

Now the general rules are:
`sponsored` for advertisements or paid placement
`ugc` for user generated content
`nofollow` for all other links you don't fully trust.

However, just using `nofollow` is still acceptable and the new tags actually have a bit of a problem:
Google recommends to mark sponsored links with either `sponsored` or `nofollow`, but they don't have any guidelines for user generated content that contains sponsored links, so it's uncertain how should you tag links in user content that might contain said paid links (since not marking paid links as `sponsored`/`nofollow` can hurt your SEO and encourage spam), and I think the consensus is to use `nofollow` for them anyway and just add `ugc` as an additional property (that is, `rel="nofollow ugc"`).

So in conclusion, it might be nice to change `rel="nofollow"` to `rel="nofollow ugc"`, but all outbound links being just `nofollow` is fine too.

**Comment by julianlam** [comment]
@oplik0 TIL, thanks for letting me know! I will add the `ugc` keyword to the `rel` tag for this purpose.
